packages/wow-ui/src/components/Header/Header.stories.ts

```diff
@@ -0,0 +1,68 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import Header from "@/components/Header";
+
+const meta = {
+  title: "UI/Header",
+  component: Header,
+  tags: ["autodocs"],
+  parameters: {
+    componentSubtitle: "헤더 컴포넌트",
+    a11y: {
+      config: {
+        rules: [{ id: "color-contrast", enabled: false }],
+      },
+    },
+  },
+  argTypes: {
+    variant: {
+      control: {
+        type: "select",
+        options: ["username", "login", "none"],
+      },
+      table: {
+        type: { summary: "username | login | none" },
+      },
+      description:
+        '헤더 종류를 나타냅니다. variant가 "username"인 경우 오른쪽에 사용자 이름을, "login"인 경우 로그인 버튼을, "none"인 경우 아무것도 표시하지 않습니다.',
+    },
```

Comment:
p4; 아래 경고가 자꾸 떠서 [공식문서](https://storybook.js.org/docs/api/arg-types#controltype) 확인해 봤더니 문서 형식을 조금 바꿔야 할 것 같아요. 저도 다른 컴포넌트에서 수정해야 할 것 같네요 ..

<img width="497" alt="image" src="https://github.com/user-attachments/assets/a4dc7048-9ebb-43ee-9701-ad86865c30cc">

```suggestion
  variant: {
      control: {
        type: "select",
      },
      options: ["username", "login", "none"],
      table: {
        type: { summary: "username | login | none" },
      },
      description:
        '헤더 종류를 나타냅니다. variant가 "username"인 경우 오른쪽에 사용자 이름을, "login"인 경우 로그인 버튼을, "none"인 경우 아무것도 표시하지 않습니다.',
    },
```

Reply:
아하 수정해뒀습니다!